source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -547,6 +541,12 @@ export abstract class CardElement implements ICardObject {
         return -1;
     }
 
+    isDesignMode(): boolean {
+        var rootElement = this.getRootElement();
```

Comment:
>var [](start = 8, length = 3)

should this be `let`?

Reply:
Should be let pretty much everywhere :) I'll change all the vars to lets later in a dedicated PR.